README.md

```diff
@@ -8,244 +8,22 @@ Build ([master](https://github.com/aws/aws-toolkit-vscode/tree/master) branch)|[
 
 The AWS Toolkit for Visual Studio Code is an extension for working with AWS services such as AWS Lambda.
 
-The toolkit is in preview and only available if built from source. This is pre-release software and we recommend against using it in a production environment.
+The toolkit is in developer preview and is available from the [Visual Studio marketplace](https://marketplace.visualstudio.com/itemdetails?itemName=AmazonWebServices.aws-toolkit-vscode). This is pre-release software and we recommend against using it in a production environment.
 
 This is an open source project because we want you to be involved. We love issues, feature requests, code reviews, pull requests or any positive contribution.
 
-## Getting Started
+## User Guide
 
-### Install the AWS SAM CLI
+The [User Guide](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/welcome.html) contains instructions for **installing**, **configuring**, and **using** the toolkit.
 
-Follow the instructions [here](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) to install the AWS SAM CLI.
+## Debugging from the Debug panel
 
-### Install the toolkit
+You can debug your Serverless Application's Lambda Function locally using the CodeLens links above the lambda handler. If you would like to use the Debug Panel to launch the debugger instead, use one of the following links to configure your project's Debug Configuration.
```

Comment:
Consider:

CodeLenses appear above Serverless Application Lambda Functions, allowing you to locally debug the Function. If you would like a Debugging experience that is [more integrated with VS Code](https://code.visualstudio.com/docs/editor/debugging), the following links can help you set up Debug Configurations for your project.

Reply:
I'm not sure about calling it 'more integrated'. This reads like the debugging experience itself is different, rather than simply being an alternate trigger for the same experience. How about:

> CodeLenses appear above Serverless Application Lambda Functions, allowing you to locally debug the Function. If you would like more control over the debugging experience, the following links can help you set up Debug Configurations for your project.

Reply:
✔️